### PR TITLE
Score Visualization

### PR DESCRIPTION
**Pull request by lucien386**

### CODE REVIEW

File: tests/test_image.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from music21 import *
+
+def test_png():
+    a = converter.parse('tests/sample.musicxml').write('lilypond.png', 'tests/result')
+    assert 1 == 1
```
Comment (xicheng87) [inline tests/test_image.py:5]: This line seems meaningless ???

File: .travis.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ before_install:
 - pip install -U pip
 - pip install -U pytest pytest-cov
 - pip install codecov
+- sudo apt-get install lilypond
```
Comment (xicheng87) [inline .travis.yml:10]: Don't you also need to install music21 as well?

Comment (lucien386) [reply .travis.yml:10]: music21 is a python package; will be installed within requirement.txt

File: tests/test_image.py (hunk 1 above)
Comment (lucien386) [reply tests/test_image.py:5]: will add more test later

**Pull request merged**